src/akamai/js/akamaiHD_flash.js

```diff
@@ -583,6 +601,37 @@ require("../../../html5-common/js/utils/constants.js");
         width: event.eventObject.width,
         height: event.eventObject.height,
       }
+        var objectHeight = event.eventObject.height;
+        var objectWidth = event.eventObject.width;
+        var wrapperHeight = $(videoItem).parent().height();
+        var wrapperWidth = $(videoItem).parent().width();
+
+        $(videoItem).css("height", objectHeight+'px');
+        var resizeStyles = {};
+
+        if (wrapperHeight > (objectHeight*1.1)) {
```

Comment:
Arbitrary attempt to avoid rounding errors as the number jumped around. Changed it to 1.05.
There's a fine line somewhere where they are too close and it intermittently won't trigger.